.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       IS_MERGE: ${{ github.base_ref == '' }}
       PREV_SHA: ${{ github.event.before }}
       TARGET_BRANCH: ${{ github.base_ref }}
+      FALLBACK_BRANCH: 'origin/develop'
```

Comment:
Is this needed since we have [this](https://github.com/secroots/dradis-pro/blob/981dad091512db7e5fcd3790999857216bf8c854/bin/rubocop-ci#L9) check in rubocop-ci